packages/mdc-elevation/_variables.scss

```diff
@@ -116,6 +116,14 @@ $mdc-elevation-ambient-map: (
 // example in a `will-change` rule.
 $mdc-elevation-property: box-shadow !default;
 
+// The default color for the elevation overlay.
+$mdc-elevation-overlay-color: #fff;
```

Comment:
I think this should be `surface`.

Reply:
It's going to be the inverse of `surface`. In dark theme, surface becomes `#121212` but the overlay should be `white`.